tools/csv-merger/csv-merger.go

```diff
@@ -405,6 +415,9 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 		},
 	}
 
+	ocsCSV.Annotations["operators.operatorframework.io/internal-objects"] = ""
+	nonInternalAPIs := []string{"storageclusters.ocs.openshift.io", "backingstores.noobaa.io", "bucketclasses.noobaa.io"}
```

Comment:
Rename this to `exposedAPIs` and make it a global var or const with each string in its own line. This will make it easier to see change sin the future.